sanctions/apps/sanctions/utils.py

```diff
@@ -1,3 +1,80 @@
 """
 Helpers for the sanctions app.
 """
+import logging
+from urllib.parse import urlencode
+
+import requests
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class SanctionsClient:
```

Comment:
i'm curious about your thoughts regarding following the api_client module pattern, like:  https://github.com/edx/subscriptions/tree/main/subscriptions/apps/api_client 

i make this suggestion, because we are also going to need an lms_client as well for the deactivation logic

i'm thinking something like this:
```
sanctions/apps/api_client/sdn_client.py
sanctions/apps/api_client/lms_client.py  (someday soon)
```

Reply:
and relatedly: I think this should probably be called `SDNClient`, given its config is specifically related to calling the SDN endpoint

Reply:
That's a good idea, that was a good pattern in subscriptions. Will modify. 
As to naming it `SDNClient`, sounds good, I initially had it as SDN. I am not sure there is a line when we chose to use the term SDN vs. sanctions even for SDN related things (like the fallback metadata and data models), since they are specific to SDN but I guess we can say that for everything in this repo right now.